internal/proto/data_coord.proto

```diff
@@ -389,9 +389,8 @@ message ChannelWatchInfo {
 
 enum CompactionType {
   UndefinedCompaction = 0;
-  InnerCompaction = 1;
-  MergeCompaction = 2;
-  MixCompaction = 3;
+  MergeCompaction = 1;
+  MixCompaction = 2;
```

Comment:
`1` shall be left there to be compatible with older version.
We could just let enum value 1 to be reserved
[Reference](https://developers.google.com/protocol-buffers/docs/proto3#reserved_values)
``` protobuf
enum CompactType{
    reserved 2;
}
```

Reply:
Thanks for your point out!